src/main/java/com/next/step/step3/validate/InputValidation.java

```diff
@@ -19,4 +25,15 @@ public void validateNumOfAttempts(int numOfAttempts) {
             throw new RuntimeException("최소 1번의 경기는 진행되어야 합니다.");
         }
     }
+
+    public void validateCarNameLength(List<String> carNames) {
+        for (String carName : carNames) {
+            if (carName.length() > MAX_CAR_NAME_LENGTH) {
+                throw new RuntimeException("자동차 이름의 최대 길이는 5글자입니다.");
+            }
```

Comment:
```suggestion
            if (carName.length() > MAX_CAR_NAME_LENGTH) {
                throw new IllegalArgumentException("자동차 이름의 최대 길이는 5글자입니다.");
            }
```
- 보다 명확한 의미를 가진 예외로 처리하는 것은 어떨까요